src/include/sof/audio/component.h

```diff
@@ -715,6 +714,7 @@ void sys_comp_kpb_init(void);
 void sys_comp_multiband_drc_init(void);
 void sys_comp_selector_init(void);
 
+void sys_comp_module_crossover_interface_init(void);
```

Comment:
seems this is not must, check up_down_mixer.c, it can be initialized with:
SOF_MODULE_INIT(up_down_mixer, sys_comp_module_up_down_mixer_interface_init);

Reply:
Statically linked testbench needs this header, it's not using the advanced component introduce and initialize system.

Reply:
@btian1 I think the SOF_MODULE_INIT is in crossover.c -- looks correct o me.